downstairs/src/lib.rs

```diff
@@ -3732,6 +3762,7 @@ pub async fn start_downstairs(
          * multiple Upstairs connecting but only one active one.
          */
         info!(log, "downstairs listening on {}", listen_on);
+        let mut id = ConnectionId(0);
```

Comment:
We already have a connection counter in ds.dss.add_connection()
Could we re-use that counter here?  Or would that complicate things
because it live in the stat code?

Reply:
I don't feel too strongly about it, but am leaning towards keeping it separate.  For example, I could imagine adding a "reset stats" function at some point in the future, which would then break `ConnectionId` uniqueness if we using that as the connection counter.

Reply:
Sure, that's fine.